freqtrade/tests/test_persistence.py

```diff
@@ -122,7 +122,7 @@ def test_update_with_bittrex(limit_buy_order, limit_sell_order):
         pair='BTC_ETH',
         stake_amount=0.001,
         fee=0.0025,
-        exchange=Exchanges.BITTREX,
+        exchange='binance',
```

Comment:
Not in scope of this PR but generally in this whole ´ccxt´ work that how do we organize our tests and which one is the "default" exchange..

Reply:
Good point - i'll revert this to bittrex for now (so bittrex is the default testing exchange throut the tests).

Would it make sense to have a variable in conftest defining this? so we can farily simply support testing over multiple exchanges?